src/objects/zcl_abapgit_objects.clas.abap

```diff
@@ -956,6 +954,15 @@ CLASS zcl_abapgit_objects IMPLEMENTATION.
       ENDLOOP.
     ENDIF.
 
+    "ignore objects that where deleted remotely
+    "but not class includes with separate file which are cleaned up during deserialize
+    LOOP AT rt_results REFERENCE INTO lr_object WHERE rstate = zif_abapgit_definitions=>c_state-deleted.
+      lv_tabix = sy-tabix.
+      IF zcl_abapgit_oo_base=>is_part_of_class( lr_object->filename ) = abap_false.
```

Comment:
this seems leaky, ideally, this method should be independent of the object type

is `lr_object->obj_type` and `lr_object->obj_name` filled with the correct information at this time?

Reply:
yes, we could add `obj_type = 'CLAS'` to the 'if'. 

it's another case that should go on this list https://github.com/abapGit/abapGit/issues/4187 as `IS_PART_OF_OBJECT` or something like it.

Reply:
I removed the is_part logic, so the behavior described in #1066 remains